docs/examples/general/data_loading/numpy_reader.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<p>I would maybe add a sentence or two here about what GDS offers for the user (so straight to GPU loading).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>